lib/shopify_cli/identity_auth.rb

```diff
@@ -255,15 +255,15 @@ def challange_params
     def auth_url
       if Environment.use_local_partners_instance?
```

Comment:
Hopefully this can simply go away altogether once spin fully replaces dev. For now I'd rather keep the changes in other folks code to spin related lines.